### ISSUE TITLE
Cannot find module 'typescript' Require stack:

### ISSUE THREAD

**Issue by blazzjosh** [kbrgl/svelte-french-toast#2]
Trying to use the lib - but on import error says - node_modules/svelte-french-toast/components/ToastIcon.svelte - Cannot find module 'typescript'

**Comment by blazzjosh** [comment]
using with sveltekit...

**Comment by kbrgl** [comment]
Could you share some code that reproduces the bug, as well as the version of Svelte French Toast you’re using? Are you facing it during development or build, or both?

As a quick check, try installing Typescript and see if that fixes it.

**Comment by blazzjosh** [comment]
the import line triggers this error-  ```import toast, {Toaster} from 'svelte-french-toast'```

Error:
```/Users/mac/Downloads/webapps/451/node_modules/.pnpm/svelte-french-toast@1.0.2_svelte@3.48.0/node_modules/svelte-french-toast/components/LoaderIcon.svelte
      at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)```

I am using "svelte-french-toast": "^1.0.2",  also not using typescript for my sveltekit installation -

**Comment by kbrgl** [comment]
These errors aren't enough to debug the issue, unfortunately. I need code (a repository or CodeSandbox/Glitch) that reproduces it.

Try installing Typescript, regardless of whether you're using it. I'm wondering if maybe it needs to be added as a peer dependency.

**Comment by kbrgl** [comment]
Fixed in 1.0.3. Upgrade your svelte-french-toast installation to the latest version to solve the issue.
